src/map/entities/battleentity.cpp

```diff
@@ -313,6 +313,12 @@ int16 CBattleEntity::GetWeaponDelay(bool tp)
     TracyZoneScoped;
     if (StatusEffectContainer->HasStatusEffect(EFFECT_HUNDRED_FISTS) && !tp)
     {
+        if (this->objtype == ENTITYTYPE::TYPE_MOB)
+        {
+            // Captures show mobs swing around 700 delay under hundered fists
```

Comment:
There's a typo here, `hundered ` -> `hundred`
that should be a justfiable commit amend to kick CI haha

Reply:
> There's a typo here, `hundered ` -> `hundred` that should be a justfiable commit amend to kick CI haha

Ok, should be good now. Thanks!